lib/addons/argocd/index.ts

```diff
@@ -1,14 +1,13 @@
+import bcrypt = require('bcrypt')
 import { Construct } from "@aws-cdk/core";
 import { HelmChart, KubernetesManifest, ServiceAccount } from "@aws-cdk/aws-eks";
 import { ManagedPolicy } from "@aws-cdk/aws-iam";
-import { ApplicationRepository, ClusterAddOn, ClusterPostDeploy, ClusterInfo, Team } from "../../spi";
 
 import { getSecretValue } from '../../utils/secrets-manager-utils';
 import { sshRepoRef, userNameRepoRef } from './manifest-utils';
 import { btoa } from '../../utils/string-utils';
-import bcrypt = require('bcrypt');
 import { Constants } from "..";
-
+import * as spi from "../../spi";
```

Comment:
Minor: What is the incentive to use wildcard import with prefix vs clear imports? Is it a number of objects in import? Namespacing?
I was going with 5 imports before using wildcards.

Let's agree and make a lint rule for this.

Reply:
lol I had the same comment on your pr today let's agree tomorrow.